scripts/buildorder.py

```diff
@@ -385,16 +437,25 @@ def main():
             packages_directories.insert(0, os.path.dirname(package))
     pkgs_map = read_packages_from_directories(packages_directories, fast_build_mode, full_buildorder)
 
+    if get_list_cirdep:
+        pkgname = None
+        if not full_buildorder:
+            pkgname = package.split("/")[-1]
+            if "gpkg" in package.split("/")[-2].split("-") and not has_prefix_glibc(pkgname):
+                pkgname += "-glibc"
+        get_list_cyclic_dependencies(pkgs_map, pkgname=pkgname, build_mode=not(fast_build_mode))
+        sys.exit(0)
+
     if full_buildorder:
         build_order = generate_full_buildorder(pkgs_map)
     else:
         build_order = generate_target_buildorder(package, pkgs_map, fast_build_mode)
 
     for pkg in build_order:
-        pkg_name = pkg.name
-        if termux_global_library == "true" and termux_pkg_library == "glibc" and "glibc" not in pkg_name.split("-"):
-            pkg_name = add_prefix_glibc_to_pkgname(pkgname)
-        print("%-30s %s" % (pkg_name, pkg.dir))
+        pkgname = pkg.name
```

Comment:
This pkg_name -> pkgname fix is not mentioned in changelog, and can be a separate commit to simplify review and future rebase/reverts/bisects.